pkg/cluster/cluster.go

```diff
@@ -89,7 +90,10 @@ type Cluster struct {
 	processMu        sync.RWMutex // protects the current operation for reporting, no need to hold the master mutex
 	specMu           sync.RWMutex // protects the spec for reporting, no need to hold the master mutex
 	ConnectionPooler map[PostgresRole]*ConnectionPoolerObjects
+	EBSVolumes       map[string]volumes.VolumeProperties
```

Comment:
```suggestion
	EBSVolumes       map[string]*volumes.VolumeProperties
```
not sure, but since we had a similar case with PreparedDatabases recently, maybe this should be with pointer?

Reply:
Nope, Cluster object is never generated anywhere. No pointer needed here.

The pointer non pointer is for K8s API/Client objects.